src/extensions/default/AutoUpdate/main.js

```diff
@@ -702,6 +709,13 @@ define(function (require, exports, module) {
             break;
         }
         console.log("AutoUpdate : Clean-up failed! Reason : " + descriptionMessage + ".\n");
+        HealthLogger.sendAnalyticsData(
+                autoUpdateEventNames.AUTOUPDATE_CLEANUP_FAILED,
+                "autoUpdate",
+                "cleanUp",
+                "fail",
+                descriptionMessage
```

Comment:
This descriptionMessage should also be added to Strings, otherwise it will never be localized